### ISSUE TITLE
Cannot move focus to facet category headers to open/close

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/earthworks#1260]
Add the ability to move focus to facet category headers and allow users to open/close each facet category using the keyboard.

**Comment by peetucket** [comment]
Consider doing in geoblacklight, not in earthworks

**Comment by peetucket** [comment]
Works in searchworks, so may be ok in blacklight.   Look at potential overrides in geoblacklight causing an issue.

tabindex should be set to a reasonable value

**Comment by peetucket** [comment]
Investigating now.  This seems to be occurring in the main branch of blacklight right now

**Comment by peetucket** [comment]
Turns out focus is being set, it's just not visible since there is no border color when we are focused.  See https://stanfordlib.slack.com/archives/C0AAK1HNY/p1724799431275469

There may be two actions needed:
1. Set a specific color for Earthworks in the CSS
2. Set a default color in Blacklight so other apps will at least inherit something (instead of blank)

**Comment by jcoyne** [comment]
I think Niqui is working on a fix in component library.

**Comment by peetucket** [comment]
Needed another update...will be fixed by https://github.com/sul-dlss/component-library/pull/37 when it's merged, a new release and cut, and Earthworks is updated to use the new release.  

Here is how it will look (tested in localhost manually):

![Image](https://github.com/user-attachments/assets/a230d53a-eabb-47a6-a3e9-8742b953c0ab)
![Image](https://github.com/user-attachments/assets/d89a5001-0cee-403d-a489-762648d88de4)
